modules/calib/include/opencv2/calib.hpp

```diff
@@ -1127,6 +1127,41 @@ CV_EXPORTS_W double stereoCalibrate( InputArrayOfArrays objectPoints,
                                      OutputArray perViewErrors, int flags = CALIB_FIX_INTRINSIC,
                                      TermCriteria criteria = TermCriteria(TermCriteria::COUNT+TermCriteria::EPS, 30, 1e-6) );
 
+// TODO: add documentation for this
+CV_EXPORTS_AS(stereoCalibrateExtrinsicExtended)  double stereoExtrinsicCalibrate( InputArrayOfArrays objectPoints,
```

Comment:
Stereo is bad choice for the function name:
- You cannot get real stereo if cameras have different type.
- The function just registers two cameras, but does not run intrinsics tuning.

I propose to name it `registerCameras`.